todo_app/app.py

```diff
@@ -1,15 +1,29 @@
-from flask import Flask
-
+from types import GetSetDescriptorType
+from flask import Flask,escape,request,render_template,redirect
+from flask.helpers import url_for
+from flask.wrappers import Request
+from todo_app.data.session_items import _DEFAULT_ITEMS, add_item,get_items
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
-
-@app.route('/')
+@app.route('/', methods=['GET', 'POST'])
 def index():
-    return 'Hello World!'
+    if request.method == "POST":
+       return redirect(url_for('addItm'))
+    else:
+        return render_template('index.html',items=get_items())
 
+@app.route('/additem',methods=['GET','POST'])
+def addItm():
```

Comment:
Not sure if this is a typo or intentional, but generally it's worth keeping these variable names fully spelled out for readability